app.py

```diff
@@ -58,7 +61,7 @@ def canary():
     return jsonify({'messages': messages}), status_code
 
 
-def get_clearpass_token(client_id, client_secret):
+def get_clearpass_token(client_id, client_secret) -> str:
```

Comment:
I recommend you add a type of string to `client_id` and `client_secret`.

Reply:
I think we actually want something like:
```
def get_clearpass_token(client_id:str, client_secret:str)
```

Reply:
Already added:
![image](https://user-images.githubusercontent.com/4566388/87168744-92ad4a00-c294-11ea-9514-fa96eecf46d9.png)

Reply:
For some reason, Github leaves the old view of the code and marks it "outdated" instead of updating it....